elements/bulbs-video/components/revealed.js

```diff
@@ -202,7 +202,7 @@ export default class Revealed extends React.Component {
   }
 
   makeVideoPlayer (element, videoMeta) {
-    element.id = videoMeta.gaPrefix;
+    element.id = jwPlayerIdCounter++;
```

Comment:
Possibly a mute point, but we really should be prefixing this instead of generically using integer ids, which I feel like could easily collide with other elements eventually.  Instead of ids `0`, `1`, `2`... it makes more sense to do something like `jw-player-0`, `jw-player-1`, `jw-player-2`